metadata.json

```diff
@@ -42,7 +42,7 @@
     },
     {
       "name": "puppet/redis",
-      "version_requirement": ">= 5.0.0 < 6.0.0"
```

Comment:
This is already in master so should be rebased. My merge script (I'll share it soon) only allows fast forwards to ensure nothing is missed.

Reply:
Rebased